src/style.css

```diff
@@ -0,0 +1,145 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 200px;
+  min-height: 404px;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+}
+
+.card__img {
+  width: 160px;
+  height: 134px;
+}
+
+.main-text {
+  font-size: 12px;
+  line-height: 18px;
+  margin: 40px 0 0;
+  font-weight: 500;
+  color: #060b35;
+}
+
+.main-text__name {
+  margin: 0;
+  margin-bottom: 4px;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+}
+
+.main-text__number {
+  margin-top: 4px;
+  margin-bottom: 16px;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+}
+
+.card__reviews {
+  height: 16px;
+  width: 166px;
+  margin-bottom: 24px;
+  display: flex;
+  flex-direction: raw;
+  align-items: flex-end;
+  justify-content: space-between;
+}
+
+.stars {
+  display: flex;
+  flex-direction: raw;
+  align-items: flex-end;
+  justify-content: space-between;
+}
+
+.stars-item {
+  margin-right: 4px;
+  background: url(\images\star-active.svg) no-repeat center;
+  height: 16px;
+  width: 16px;
+}
+
+.stars-inactive {
+  height: 16px;
+  width: 16px;
+  margin-right: 0;
+
+  background: url(\images\star.svg) no-repeat center;
+  background-repeat: no-repeat;
+}
```

Comment:
Split your css code to different files - BEM blocks.

According to [checklist](https://github.com/mate-academy/layout_product-cards/blob/master/checklist.md)

`11.[FILE STRUCTURE] - You should create separate file for each BEM block`

detailed info how to do it in [this video](https://mate.academy/learn/css-advanced/bem#/video/917)